src/content/academic-grants-2022.mdx

```diff
@@ -0,0 +1,205 @@
+---
+metaTitle: Academic Grants Round 2022
+metaDescription: The Ethereum Foundation is sponsoring a wave of grants to support Ethereum-related academic work. Find all the details you need to apply here.
+metaImage: /images/academic-grants-hero.png
+isClosed: true
+---
+
+**We're calling all those interested in expanding academic knowledge throughout the Ethereum ecosystem!**
```

Comment:
This bold section at the top looks different between prod and this PR 
![Screenshot 2024-03-16 at 12 00 59 PM](https://github.com/ethereum/esp-website/assets/15589226/231f25e2-edf5-46ea-a981-b0878931831b)

Reply:
Correct. I've answered this also in the prev comment but in short, now every heading in the md file will appear in the sidebar, so we will need to tweak it a bit to avoid that.

Reply:
I've added a `Summary` component + a `hasSummary` frontmatter prop to display the summary link in the sidenav 4fa80692a736cf4fb72c0c98804a8e54a80f07a4